.gitlab/functional_test/system_probe.yml

```diff
@@ -76,7 +76,6 @@ kitchen_test_system_probe_windows_x64:
   variables:
     KITCHEN_ARCH: x86_64
     KITCHEN_OSVERS: "win2012r2"
-    CHEF_VERSION: 14.12.9
```

Comment:
Is this change wanted? What version is this going to use by default?

Reply:
The version defined in `.kitchen_test_system_probe`: `15.16.4`

Reply:
It is required for the `azure?` check here: https://github.com/DataDog/datadog-agent/pull/11071/files#diff-f3d6afcce67313e76a58c652295befb80079ecff110015a64fa5a62e99ff6d5aR8